projects/plugins/jetpack/extensions/store/wordpress-com/reducer.ts

```diff
@@ -32,6 +32,7 @@ const INITIAL_STATE: PlanStateProps = {
 			_meta: {
 				isRequesting: false,
 			},
+			nextTier: null,
```

Comment:
Should define a default value here?

```suggestion
			nextTier: {
				slug: '',
				value: 20,
				limit: 20,
				redeableLimit: '',
			},
```

Reply:
I was still wondering if we should. Null seems like a good option. But in any case, that was for a follow up as I'm going to be shaping both current and next tier props